### DISCUSSION TITLE
importing seed points

### DISCUSSION THREAD

**Discussion by felix0685** [neperfepx/neper#362] [Q&A] [answered]
Hello,

I am wondering if there is a possibility to import seed points into Neper and then let Neper do a tesselation.

Background is that we want to create some Micro structures where we mix columnar grains and regular or columnar grains that are rotated to each other (i. e. 3D Printing with alternating scanning direction, so the grains would follow the X Axis for some layers and then the scanning direction would be changed so it follows the Y-direction...)

our idea was to define the seed points with X/Y/Z-coordinates and then have Neper create a microstructure.

is there a way to do this?

**Comment by rquey** [comment, marked as answer]
Yes, use `-morphooptiini`.
